infrastructure/environments/prod/main.tf

```diff
@@ -80,5 +78,5 @@ module service_bus {
   environment         = var.ENVIRONMENT
   location            = var.REGION
   name                = local.service_bus_name
-  resource_group_name = azurerm_resource_group.products.name
+  resource_group_name = data.azurerm_resource_group.products.name
```

Comment:
It'd be good to add the search service sku into the `<environment>/main.tf` files, setting it to `standard` for prod.

Reply:
good idea. Done